kubeflow/trainer/utils/utils_test.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2025 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from kubeflow.trainer.constants import constants
+from kubeflow.trainer.types import types
+from kubeflow.trainer.utils import utils
+from kubeflow.trainer.test.common import TestCase, SUCCESS, FAILED
+
+def _build_runtime() -> types.Runtime:
+    runtime_trainer = types.RuntimeTrainer(
+        trainer_type=types.TrainerType.CUSTOM_TRAINER,
+        framework="torch",
+        device="cpu",
+        device_count="1",
+    )
+    runtime_trainer.set_command(constants.DEFAULT_COMMAND)
+    return types.Runtime(name="test-runtime", trainer=runtime_trainer)
+
+
+
+@pytest.mark.parametrize(
+    "test_case",
+    [
+        TestCase(
+            name="with args dict always unpacks kwargs",
+            expected_status=SUCCESS,
+            config={
+                "func": (lambda: print("Hello World")),
+                "func_args": {"batch_size": 128, "learning_rate": 0.001, "epochs": 20},
+                "runtime": _build_runtime(),
+            },
+            expected_output={
+                "contains": [
+                    "<lambda>(**",
+                    str({"batch_size": 128, "learning_rate": 0.001, "epochs": 20}),
+                ],
+            },
```

Comment:
Please can you follow the same output as here: https://github.com/kubeflow/sdk/blob/main/kubeflow/trainer/utils/utils_test.py#L46C13-L56 ?
Just define the complete output in the `expected_output` argument and assert it with returns from get_command_using_train_func().
```suggestion
            expected_output=[
                    "bash", "-c" "torchrun", "<lambda>(**....")", ....
            ]
```